lib/screens/profile/add_friends.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:eventure/providers/user_provider.dart';
+import 'package:eventure/widgets/inputs/custom_input_line.dart';
+import 'package:eventure/widgets/widgets.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../../models/user.dart';
+
+class AddFriendsScreen extends StatefulWidget {
+  const AddFriendsScreen({super.key});
+  @override
+  _AddFriendsScreenState createState() => _AddFriendsScreenState();
+}
+
+class _AddFriendsScreenState extends State<AddFriendsScreen> {
+  @override
+  void initState(){
+    super.initState();
+    context.read<UserProvider>().fetchUsersStartingWith();
+  }
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
Wegen dem Design des Screen guck dir vielleicht einfach mal den Design PR an bzw. warte bis der fertig und in main ist, um dann die jeweiligen Änderungen auch bei dir zu haben

Reply:
Der neuste Stand vom Design ist auf main. Einfach mal schauen, dass du Hintergrund und Farben usw. daran anpasst